instance/models/mixins/database.py

```diff
@@ -92,6 +92,7 @@ def _create_database(cursor, database):
     with warnings.catch_warnings():
         warnings.simplefilter("ignore")
         cursor.execute('CREATE DATABASE IF NOT EXISTS `{db}` DEFAULT CHARACTER SET utf8'.format(db=database))
+        cursor.execute('USE `{db}`; CREATE TABLE IF NOT EXISTS `opencraft` (col INT)'.format(db=database))
```

Comment:
@xirdneh, @giovannicimolin, should we be adding code that is only needed for the integration tests here?

Reply:
@john2x @giovannicimolin I believe there's only two changes outside of tests. The first one adds the ability to specify the prefix used in MySQL usernames. It is only used in tests and if that parameter is not specified then creating an instance will work in the same way 
The second one is about adding an empty table, which I think it'll help us outside of integration tests. Because Ocim creates empty databases in the beginning, so if anything goes wrong it's difficult to craft a query which will return this unused databases.
We could also remove these changes but they won't break any current behavior. This was tested by spinning up a new app server in stage, too.

Reply:
@xirdneh, can't we create this database in the integration test itself at a later stage?

Reply:
@lgp171188 That would defeat the purpose of creating that database because something else could go wrong in between and we'd still have an empty database which are harder to delete.
But, we could probably add it to the integration playbook although that's outside of the scope of this ticket.
Would you like me to delete this and create a ticket to add it to the integration playbook or find another solution?

Reply:
>That would defeat the purpose of creating that database because something else could go wrong in between and we'd still have an empty database which are harder to delete.
But, we could probably add it to the integration playbook although that's outside of the scope of this ticket.
Would you like me to delete this and create a ticket to add it to the integration playbook or find another solution?

@xirdneh, it would be better to do it in the code/playbook related to the integration tests. You can create a new ticket for doing that and leave this change out of this PR.